src/auth/components/SignInForm/style.less

```diff
@@ -42,17 +42,19 @@
           border: none;
           flex-grow: 1;
           font-size: 16px;
+          background-color: var(--theme-background);
+          color: var(--theme-text-on-background-1);
```

Comment:
is this needed? (see above comment)

Reply:
Required for sign in text boxes to match background color and text to be readable.